examples/AvroSpecific/User.avsc

```diff
@@ -1,15 +1,10 @@
-﻿{"namespace": "Confluent.Kafka.Examples.AvroSpecific",
- "type": "record",
- "name": "User",
- "fields": [
-     {"name": "name", "type": "string"},
-     {"name": "favorite_number",  "type": ["int", "null"]},
-     {"name": "favorite_color", "type": ["string", "null"]},
-     {"name": "hourly_rate", "type": {
```

Comment:
it's kind of a shame to get rid of the decimal example.
i guess it's not a big deal though, interoperability with the other examples is probably more important.

Reply:
I've kept it, now it's an optional field with a default value so it's compatible with the others